packages/core-api/src/app/index.ts

```diff
@@ -16,3 +16,4 @@
 
 export { useApp } from './AppContext';
 export * from './types';
+export * from './configLoaders';
```

Comment:
I'm thinking if we're going to go with this approach then maybe a structure like this:
```
packages/core-api/src/app/config/
  index.ts
  composeConfigLoaders.ts
  loaders/
    index.ts
    windowConfigLoader.ts
    envConfigLoader.ts (?)
    defaultConfigLoader.ts (?)
```